### ISSUE TITLE
Unterminated string constant

### ISSUE THREAD

**Issue by antonioribeiro** [vuejs/vue-loader#1453]
### Version
15.4.2

### Reproduction link
[https://github.com/antonioribeiro/eventos](https://github.com/antonioribeiro/eventos)

### Steps to reproduce
Add a line break to a v-model:
``` 
v-model="
   subEvents.form.fields.associated_subevent_id

"
``` 
And you should get the error. 

The problem is that I cannot reproduce the error in a minimal environment, the error did not happen, but the project we are building is Open Source, so you can clone it and just: 

- git clone https://github.com/antonioribeiro/eventos
- change any v-model of whatever file in the directory `resources/js/views`, to break a v-model in more than one line
- npm install
- npm run dev

And it should break

### What is expected?
It to work...?

### What is actually happening?
This error:

```
 ERROR in ./resources/js/views/SubEventsForm.vue?vue&type=template&id=5fe8fece& (./node_modules/vue-loader/lib/loaders/templateLoader.js??vue-loader-options!./node_modules/vue-loader/lib??vue-loader-options!./resources/js/views/SubEventsForm.vue?vue&type=template&id=5fe8fece&)
Module build failed (from ./node_modules/vue-loader/lib/loaders/templateLoader.js):
SyntaxError: Unterminated string constant (2:98)
    at Parser.pp$4.raise (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:2610:13)
    at Parser.pp$8.readString (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:3436:35)
    at Parser.pp$8.getTokenFromCode (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:3220:17)
    at Parser.pp$8.readToken (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:2975:15)
    at Parser.readToken (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:4398:22)
    at Parser.pp$8.nextToken (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:2966:15)
    at Parser.pp$8.next (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:2923:8)
    at Parser.pp.eat (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:569:10)
    at Parser.pp$3.parsePropertyValue (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:2309:12)
    at Parser.parseObj (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:4472:14)
    at Parser.pp$3.parseExprAtom (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:2077:17)
    at Parser.parseExprAtom (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:4372:24)
    at Parser.pp$3.parseExprSubscripts (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:1955:19)
    at Parser.pp$3.parseMaybeUnary (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:1932:17)
    at Parser.pp$3.parseExprOps (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:1874:19)
    at Parser.pp$3.parseMaybeConditional (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:1857:19)
    at Parser.pp$3.parseMaybeAssign (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:1832:19)
    at Parser.pp$3.parsePropertyValue (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:2310:87)
    at Parser.parseObj (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:4472:14)
    at Parser.pp$3.parseExprAtom (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:2077:17)
    at Parser.parseExprAtom (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:4372:24)
    at Parser.pp$3.parseExprSubscripts (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:1955:19)
    at Parser.pp$3.parseMaybeUnary (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:1932:17)
    at Parser.pp$3.parseExprOps (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:1874:19)
    at Parser.pp$3.parseMaybeConditional (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:1857:19)
    at Parser.pp$3.parseMaybeAssign (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:1832:19)
    at Parser.pp$3.parseExprList (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:2528:20)
    at Parser.pp$3.parseSubscripts (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:1983:29)
    at Parser.pp$3.parseExprSubscripts (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:1958:21)
    at Parser.pp$3.parseMaybeUnary (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:1932:17)
    at Parser.pp$3.parseExprOps (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:1874:19)
    at Parser.pp$3.parseMaybeConditional (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:1857:19)
    at Parser.pp$3.parseMaybeAssign (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:1832:19)
    at Parser.pp$3.parseExprList (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:2528:20)
    at Parser.pp$3.parseExprAtom (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:2073:26)
    at Parser.parseExprAtom (/Users/antoniocarlosribeiro/code/alerj/eventos/node_modules/vue-template-es2015-compiler/buble.js:4372:24)
 @ ./resources/js/views/SubEventsForm.vue?vue&type=template&id=5fe8fece& 1:0-217 1:0-217
 @ ./resources/js/views/SubEventsForm.vue
 @ ./resources/js/routes.js
 @ ./resources/js/pages/app.js
 @ ./resources/js/pages/boostrap.js
 @ ./resources/js/app.js
 @ multi ./resources/js/app.js ./resources/sass/app.scss
```

**Comment by yyx990803** [comment]
This should have been fixed by vuejs/vue#9184

**Comment by antonioribeiro** [comment]
Indeed. Thank you.
